src/ert/ensemble_evaluator/_ensemble.py

```diff
@@ -198,11 +191,10 @@ async def send_event(
         url: str,
         event: Event,
         token: str | None = None,
-        cert: str | bytes | None = None,
         retries: int = 10,
     ) -> None:
-        async with Client(url, token, cert, max_retries=retries) as client:
-            await client._send(event_to_json(event))
+        async with Client(url, token) as client:
+            await client._send(event_to_json(event), retries)
```

Comment:
Why do we want to move the `retries`-parameter from the client constructor to the send method?

Reply:
Two reasons.
1) Making it more stateless
2) I'd like to different retries for connect (which should happen fast) and for a default message sending